### PR TITLE
Feature/anandrkskd/389

### PR DESCRIPTION
**Pull request by anandrkskd**
#389  updated with Increased verbosity of `mesheryctl update` to show new docker image pull

### CODE REVIEW

File: downloadMesheryCandidate.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#echo ${A:1}
+
+
+#NAME="mesheryctl-$MESHERY_VERSION"
+URL="https://github.com/layer5io/meshery/releases/download/${MESHERY_VERSION}/mesheryctl_${MESHERY_VERSION:1}_${OSEXT}_${OSARCHITECTURE}.zip"
+#URL="https://github.com/layer5io/meshery/releases/download/v0.1.6/mesheryctl_0.1.6_Darwin_i386.zip"
+#printf "Downloading %s from %s ..." "$NAME" "$URL"
+#curl -L "$URL" | tar xz
+
+curl -L ${URL} -o /tmp/meshery.zip
```
Comment (leecalcote) [inline downloadMesheryCandidate.sh]: @anandrkskd can we use the user's current working directory as the download location?  `/tmp` isn't available on MacOS by default nor on Windows.

File: downloadMesheryCandidate.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+unzip /tmp/meshery.zip
```
Comment (leecalcote) [inline downloadMesheryCandidate.sh]: @anandrkskd what do we do in the even the user doesn't have `unzip`?

File: downloadMesheryCandidate.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/sh
```
Comment (leecalcote) [inline downloadMesheryCandidate.sh]: @anandrkskd let's use`#!/usr/bin/env bash` to improve portability

**Comment by leecalcote** [comment]
@anandrkskd will you breakout `mesheryctl/cmd/update.go` into a separate PR?

**Comment by leecalcote** [comment]
@anandrkskd I like the branch naming scheme you’ve used here. Thumbs up.

**Comment by anandrkskd** [comment]
SIr @leecalcote , I have removed the bash script for now, and this PR will be used for increased verbosity of mesheryctl update.

File: downloadMesheryCandidate.sh (hunk 2 above)
Comment (anandrkskd) [reply downloadMesheryCandidate.sh]: Sir @leecalcote  we are using zip to compress so I'm not sure if tar would work!

**Pull request merged**